### ISSUE TITLE
Not allowed to resolve a type with an XML tag, until it fully closed

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#11486]
1. Given the following code, Ctrl+. at $

``` C#
        /// <summary>
        ///     Returns a value indicating whether the current project has the specified capability
        /// </summary>
        /// <exception cref="ArgumentNullException$"
        bool Contains(string capability);
```

Expected: To be be offered to add "using System" to resolve ArgumentNullException
Actual: Not offerred

Need to completely close the XML tag before I'm allowed to resolve:

``` C#
        /// <summary>
        ///     Returns a value indicating whether the current project has the specified capability
        /// </summary>
        /// <exception cref="ArgumentNullException"></exception>
        bool Contains(string capability);
```

This is a bit annoying, because completion doesn't close it for when you choose <exception from completion window.

**Comment by Pilchie** [comment]
This is happening because the compiler doesn't emit diagnostics for the unbound type until the xml parses.  Not sure how hard it would be to change that.

**Comment by CyrusNajmabadi** [comment]
Assigning to compiler.  This seems like a standard case of semantics not being reported because of syntax errors.
